diff --git a/node_modules/expo-dev-launcher/ios/SwiftUI/DevServersView.swift b/node_modules/expo-dev-launcher/ios/SwiftUI/DevServersView.swift
index 8520f7e..7b8afd9 100644
--- a/node_modules/expo-dev-launcher/ios/SwiftUI/DevServersView.swift
+++ b/node_modules/expo-dev-launcher/ios/SwiftUI/DevServersView.swift
@@ -113,6 +113,7 @@ struct DevServersView: View {
         showingInfoDialog = true
       } label: {
         Text("info".uppercased())
+          .foregroundColor(.primary)
           #if os(tvOS)
           .font(.system(size: 24))
           #else
diff --git a/node_modules/expo-dev-launcher/ios/SwiftUI/HomeTabView.swift b/node_modules/expo-dev-launcher/ios/SwiftUI/HomeTabView.swift
index f0c619d..43b9e1f 100644
--- a/node_modules/expo-dev-launcher/ios/SwiftUI/HomeTabView.swift
+++ b/node_modules/expo-dev-launcher/ios/SwiftUI/HomeTabView.swift
@@ -51,6 +51,7 @@ struct HomeTabView: View {
         .padding()
       }
     }
+    .background()
     .overlay(
       DevServerInfoModal(showingInfoDialog: $showingInfoDialog)
     )
diff --git a/node_modules/expo-dev-launcher/ios/SwiftUI/SettingsTabView.swift b/node_modules/expo-dev-launcher/ios/SwiftUI/SettingsTabView.swift
index c4ae8d5..35d3c2c 100644
--- a/node_modules/expo-dev-launcher/ios/SwiftUI/SettingsTabView.swift
+++ b/node_modules/expo-dev-launcher/ios/SwiftUI/SettingsTabView.swift
@@ -56,6 +56,7 @@ struct SettingsTabView: View {
       .padding()
     }
     .frame(maxWidth: .infinity, maxHeight: .infinity)
+    .background()
     .navigationBarHidden(true)
   }
 
diff --git a/node_modules/expo-dev-launcher/ios/SwiftUI/UpdatesTab/UpdatesTabView.swift b/node_modules/expo-dev-launcher/ios/SwiftUI/UpdatesTab/UpdatesTabView.swift
index 7e172d9..ba7436b 100644
--- a/node_modules/expo-dev-launcher/ios/SwiftUI/UpdatesTab/UpdatesTabView.swift
+++ b/node_modules/expo-dev-launcher/ios/SwiftUI/UpdatesTab/UpdatesTabView.swift
@@ -20,6 +20,7 @@ struct UpdatesTabView: View {
         }
       }
     }
+    .background()
   }
 }
 
