diff --git a/node_modules/expo-dev-menu/ios/SwiftUI/HeaderView.swift b/node_modules/expo-dev-menu/ios/SwiftUI/HeaderView.swift
index 49279bc..90f7aec 100644
--- a/node_modules/expo-dev-menu/ios/SwiftUI/HeaderView.swift
+++ b/node_modules/expo-dev-menu/ios/SwiftUI/HeaderView.swift
@@ -30,7 +30,11 @@ struct HeaderView: View {
       } label: {
         ZStack {
           Circle()
+          #if os(tvOS)
+            .fill(Color.expoSystemGray4.opacity(0.2))
+          #else
             .fill(Color.expoSystemGray6)
+          #endif
             .frame(width: 36, height: 36)
 
           Image(systemName: "xmark")
